frontend/src/components/Schedules/SchedulePicker/SchedulePickerContainer.tsx

```diff
@@ -12,7 +15,8 @@ export default function SchedulePickerContainer({
 }: MeetingDateTime) {
   const params = useParams<{ uuid: string }>();
   const uuid = params.uuid!;
-  const { data: meetingSchedules } = useGetMyScheduleQuery(uuid);
+  const { state } = useContext(AuthContext);
+  const { data: meetingSchedules } = useGetMyScheduleQuery(uuid, state.userName);
```

Comment:
[P3]

사소하지만, 아래처럼 바꿔주시면 좋을 것 같아요 :)

```suggestion
  const { userName } = useContext(AuthContext).state;
  const { data: meetingSchedules } = useGetMyScheduleQuery(uuid, userName);
```